src/styles/blocks/_calendar

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  margin: auto;
+  padding: 30px $calendar-paddig-right 30px $calendar-paddig-left;
```

Comment:
Why do you need left and right paddings as different variables if they're the same? You could use something like
`padding: 30px $calendar-padding-horizontal`. Also you have typo in word "padding"